src/Repository/ConferenceRepository.php

```diff
@@ -85,6 +85,18 @@ public function getAllConferencesAsRow(): array
             ;
     }
 
+    /** @return array<Conference> */
+    public function getNullCoordinatesConferences(): array
+    {
+        return $this->createQueryBuilder('c')
+            ->andWhere('c.online = :falseValue')
+            ->setParameter('falseValue', false)
+            ->andWhere('c.coordinates IS NULL')
+            ->getQuery()
+            ->execute()
```

Comment:
Can you update this query to not fetch all conference at once, it will consume too much memory.
Instead you must use an iterator `$qb->getQuery()->iterate()`